### DISCUSSION TITLE
怎么在自己的项目中使用antd的 CSS-in-JS  并且可以自己添加一些token

### DISCUSSION THREAD

**Discussion by pangxzi** [ant-design/ant-design#39391] [Q&A]
怎么在自己的项目中使用antd的 CSS-in-JS  并且可以自己添加一些token

**Comment by zombieJ** [comment]
如果是组件级别的拓展可以参考一下 pro components: https://github.com/ant-design/pro-components

如果你是应用使用的话那推荐使用应用级别的 cssinjs 方案哈~
